packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -270,12 +270,25 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
       props.inputProps.onClick(ev as React.MouseEvent<HTMLInputElement>);
     }
   };
-  const _onInputChange = (value: string, composing?: boolean) => {
+  const _onInputChange = (value: string, composing?: boolean, resultItemsList?: T[]) => {
     if (!composing) {
       // update query string
       setQueryString(value);
       !isSuggestionsShown ? showPicker(true) : null;
-      onInputChange ? onInputChange(value) : null;
+      if (!resultItemsList) {
```

Comment:
I think this will always run? I think you can just declare it here so the prop can fill it in.

Reply:
i am not sure i understand.. what line are talking about :) ? we want to initialize the resiltItmsList to be an array in case it is not defined.

Reply:
Correct. What I was trying to say is that I think it will always be undefined, so this clause will always run. But TBH it's fine either way